pages/index.js

```diff
@@ -68,6 +63,104 @@ export default function HomePage() {
       };
     });
   }
+  // __________________________________________________________________
+
+  // _______________HANDLE SUBMIT _____________________________________
+
+  function handleSubmit(event) {
+    event.preventDefault();
+
+    const formData = new FormData(event.target);
+    const data = Object.fromEntries(formData);
+
+    if (data.nameInput.trim() === "") {
+      alert(
+        "Wer das hier findet, hat richtig getestet! Danke von den SurvivalPets - Please insert a Name!"
+      );
+      return;
+    }
+
+    const currentTime =
+      mode === "select" ? Date.now() / 1000 : selectedPet.createdAt;
+
+    const updatedPet = {
+      ...selectedPet,
+      name: data.nameInput,
+      lastUpdated: Date.now() / 1000,
+      createdAt: currentTime,
+    };
+    setSelectedPet(updatedPet);
+
+    setMode("livingroom");
+  }
+  // ___________________________________________________________________
+
+  // __________________________TIME LOGIC_______________________________
+
+  useEffect(() => {
+    if (selectedPet) {
+      const interval = setInterval(() => {
+        setSelectedPet((prevPet) => {
+          // calculate reduction based on elapsed time
+          const currentTime = Date.now() / 1000;
+          const elapsedTime = currentTime - prevPet.lastUpdated;
+          const reduction = Math.floor(elapsedTime) * 2; // --> parameter for stat reduction
+
+          // decrease stats over time
+
+          if (reduction > 0) {
+            if (prevPet.health === 0) {
```

Comment:
Just a comment: since you're basically doing the same thing three times here, you could probably apply an array method here. But I guess it wouldn't make your code easier to read, so just forget it. I just thought I had to comment something after reading all your code.